src/config.json

```diff
@@ -37,7 +37,7 @@
     "hazpyInitUrl": "https://raw.githubusercontent.com/nhrap-hazus/hazpy/test/hazpy/__init__.py",
     "repoZipfileUrl": "https://github.com/nhrap-hazus/HHIT/archive/test.zip",
     "toolInitUrl": "https://raw.githubusercontent.com/nhrap-hazus/HHIT/test/src/__init__.py",
-    "virtualEnvironment": "hazus_env_hhit",
```

Comment:
Was it the manage.py script that didn't allow hazus_env_hhit to work? If so, I ran into something like that with Export, and was able to make a few adjustments to get it to work. I'll dig more into this code next week, and try it out locally.

Reply:
I decided to keep using hazus_env for now as technically HHIT uses its own hazpy code and not the hazpy which is installed from anaconda.org into hazus_env.